storage/bucket.go

```diff
@@ -1904,6 +1921,38 @@ func toRPOFromProto(b *storagepb.Bucket) RPO {
 	}
 }
 
+func toBucketCustomPlacementConfig(c *raw.BucketCustomPlacementConfig) *BucketCustomPlacementConfig {
```

Comment:
For parallelism maybe let's call them `customPlacementFromRaw` and `customPlacementFromProto`?